lib/watch-sidebar.js

```diff
@@ -106,13 +105,34 @@ export default class WatchSidebar {
       name: v.getCode(),
       value: k
     }));
-    WatchesPicker.onConfirmed = item => {
-      this.watchViews[item.value].destroy();
-      this.watchViews.splice(item.value, 1);
-      if (this.watchViews.length === 0) this.addWatch();
-    };
-    WatchesPicker.setItems(watches);
-    WatchesPicker.toggle();
+    watchesPicker = new SelectListView({
+      items: watches,
+      elementForItem: watch => {
+        const element = document.createElement("li");
+        element.textContent = watch.name;
+        element.dataset = watch.name;
```

Comment:
Just the dataset line, 113 right? Left in from an example, but does not seem necessary.